### PR TITLE
DEP-219: Removed issued that have been fixed by Nuance in the webchat…

### PR DESCRIPTION
**Pull request by tonymyers**
… skin

### CODE REVIEW

File: conf/messages [hunk 1]
```diff
@@ -239,15 +236,13 @@ help.webchat.nonaccessiblecontent.h2bullets=Non compliance with the accessibilit
...
+help.webchat.howwetested.p4=This page was prepared on 05/03/2020.pass to our supplier
 
 help.webchat.whatweredoing.h2=What we''re doing to improve accessibility
-help.webchat.whatweredoing.p1=We''re working with Nuance, the new provider of our webchat, to continuously improve the quality and accessibility of this service. Our accessibility experts also regularly review the webchat and make recommendations which we then pass to our supplier Nuance. We acknowledge that some adjustments are simple and immediate, but others require deeper analysis and development and will be slower to take effect.
+help.webchat.whatweredoing.p1=We''re working with Nuance, the new provider of our webchat, to continuously improve the quality and accessibility of this service. Our accessibility experts also regularly review the webchat and make recommendations which we then pass to Nuance. We acknowledge that some adjustments are simple and immediate, but others require deeper analysis and development and will be slower to take effect.
```
Comment (oc-ste) [inline conf/messages]: did you mean to delete 

> our supplier

File: conf/messages [hunk 2]
```diff
@@ -239,15 +236,13 @@ help.webchat.nonaccessiblecontent.h2bullets=Non compliance with the accessibilit
...
-help.webchat.nonaccessiblecontent.li6=Users of Jaws may find they receive no notification that the agent has sent a message. This fails WCAG section 4.1.2 Name, Role, Value
 
 help.webchat.howwetested.h2=How we tested this webchat
-help.webchat.howwetested.p1=The service was last tested on 18/02/2020 and was checked for compliance with the WCAG 2.1 AA standard.
-help.webchat.howwetested.p2=The service was built using parts that will be tested by the Digital Accessibility Centre. The full service was tested by HMRC and included a disabled user.
-help.webchat.howwetested.p3=This page was prepared on 18/02/2020.
+help.webchat.howwetested.p1=The service was last tested on 05/03/2020 and was checked for compliance with the WCAG 2.1 AA standard.
+help.webchat.howwetested.p2=We have fixed and therefore deleted three items from the originallist.
+help.webchat.howwetested.p3=The service was built using parts that will be tested by the Digital Accessibility Centre. The full service was tested by HMRC and included a disabled user.
+help.webchat.howwetested.p4=This page was prepared on 05/03/2020.pass to our supplier
```
Comment (oc-ste) [inline conf/messages]: .pass to our supplier?

File: conf/messages [hunk 3]
```diff
@@ -239,15 +236,13 @@ help.webchat.nonaccessiblecontent.h2bullets=Non compliance with the accessibilit
...
-help.webchat.nonaccessiblecontent.li5=Jaws stops reading out the chat conversation when the user is in the message input box. Users should click ‘tab button’ and click up arrow
-help.webchat.nonaccessiblecontent.li6=Users of Jaws may find they receive no notification that the agent has sent a message. This fails WCAG section 4.1.2 Name, Role, Value
 
 help.webchat.howwetested.h2=How we tested this webchat
-help.webchat.howwetested.p1=The service was last tested on 18/02/2020 and was checked for compliance with the WCAG 2.1 AA standard.
-help.webchat.howwetested.p2=The service was built using parts that will be tested by the Digital Accessibility Centre. The full service was tested by HMRC and included a disabled user.
-help.webchat.howwetested.p3=This page was prepared on 18/02/2020.
+help.webchat.howwetested.p1=The service was last tested on 05/03/2020 and was checked for compliance with the WCAG 2.1 AA standard.
+help.webchat.howwetested.p2=We have fixed and therefore deleted three items from the originallist.
```
Comment (oc-ste) [inline conf/messages]: originallist?

File: conf/messages.cy [hunk 4]
```diff
@@ -233,16 +230,14 @@ help.webchat.nonaccessiblecontent.h2bullets=Non compliance with the accessibilit
...
-help.webchat.nonaccessiblecontent.li4=Some screen readers may fail to tell the user whether the print-end chat menu is open or not. This fails WCAG section 4.1.2 Name, Role, Value
-help.webchat.nonaccessiblecontent.li5=Jaws stops reading out the chat conversation when the user is in the message input box. Users should click ‘tab button’ and click up arrow
-help.webchat.nonaccessiblecontent.li6=Users of Jaws may find they receive no notification that the agent has sent a message. This fails WCAG section 4.1.2 Name, Role, Value
 
 help.webchat.howwetested.h2=How we tested this webchat
-help.webchat.howwetested.p1=The service was last tested on 18/02/2020 and was checked for compliance with the WCAG 2.1 AA standard.
-help.webchat.howwetested.p2=The service was built using parts that will be tested by the Digital Accessibility Centre. The full service was tested by HMRC and included a disabled user.
-help.webchat.howwetested.p3=This page was prepared on 18/02/2020.
+help.webchat.howwetested.p1=The service was last tested on 05/03/2020 and was checked for compliance with the WCAG 2.1 AA standard.
```
Comment (oc-ste) [inline conf/messages.cy]: Ensure this file lines up to the english message file, atm it does not

File: conf/messages (hunk 1 above)
Comment (tonymyers) [reply conf/messages]: No just remove the text our supplier

File: conf/messages (hunk 3 above)
Comment (tonymyers) [reply conf/messages]: I've double check and that's what the content designer wants to put in

File: conf/messages (hunk 2 above)
Comment (tonymyers) [reply conf/messages]: well spotted

File: conf/messages.cy (hunk 4 above)
Comment (tonymyers) [reply conf/messages.cy]: Done

**Pull request merged**